targets/unity-v2/Testing/Tests/PersistentSockets/PubSubTest.cs

```diff
@@ -0,0 +1,73 @@
+#if ENABLE_PLAYFABPUBSUB_API
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace PlayFab.UUnit
+{
+    public class PubSubTest : UUnitTestCase
+    {
+        static PlayFab.EventsModels.EntityKey _MyEntityKey = null;
+        static string _previousTitleId = null;
+
+        public override void Tick(UUnitTestContext testContext)
+        {
+            // No async work needed?
+            // should this be a theoretical HubConnection SendAsync call to print a message out in the test?
+        }
+        public override void TearDown(UUnitTestContext testContext)
+        {
+        }
+
+        public override void ClassTearDown()
+        {
+        }
+
+        [UUnitTest]
+        public void TestPubSubConstruction(UUnitTestContext testContext)
+        {
+            //_testContext = testContext;
+            Debug.Log("Starting PubSub UnitTest");
+
+            // specific title id for relay test
+            _previousTitleId = PlayFabSettings.TitleId;
+            PlayFabSettings.TitleId = "70B02F89";
+            PlayFabSettings.VerticalName = "spi-relay";
+
+            ClientModels.LoginWithCustomIDRequest login = new ClientModels.LoginWithCustomIDRequest();
+            login.CustomId = "ToddIsTryingToTestPersistentSocketsInUnity" + Guid.NewGuid().ToString();
```

Comment:
I'm not a fan of generating a new user every test execution. It's not going to break anything, but it's an overhead on the PlayFab system as a whole, which is unnecessary.

Reply:
Also, you probably shouldn't commit your name into source code.